Cargo.toml

```diff
@@ -11,13 +11,14 @@ git = "https://github.com/dwrensha/capnpc-rust"
 log = "*"
 rand = "*"
 uuid = "*"
+bufstream = "*"
 
 [dependencies.capnp]
 git = "https://github.com/danburkert/capnproto-rust"
 branch = "async"
 
 [dependencies.mio]
-git = "https://github.com/carllerche/mio"
+git = "https://github.com/danburkert/mio"
```

Comment:
Compatibility in the sense that the newer code would still compile for those using the old API - while having compile time warnings about the deprecation of the old functions. Just an optional step that I doubt will be implemented as mio isn't really at a _stable_ point, like Rust is.